demos/demo_teststand_calibration.cpp

```diff
@@ -50,28 +50,17 @@ static THREAD_FUNCTION_RETURN_TYPE control_loop(void* robot_void_ptr)
 {
     Teststand& robot = *(static_cast<Teststand*>(robot_void_ptr));
 
-    std::array<double, 2> zero_to_index_angle;
-    std::array<double, 2> index_angle;
-    bool mechanical_calibration;
-
-    zero_to_index_angle.fill(0.0);
-    index_angle.fill(0.0);
-    mechanical_calibration = true;
-
-    robot.calibrate(zero_to_index_angle, index_angle, mechanical_calibration);
-
-    for (unsigned i = 0; i < 2; ++i)
-    {
-        rt_printf("zero_to_index_angle[%d] = %f\n", i, zero_to_index_angle[i]);
-        rt_printf("index_angle[%d] = %f\n", i, index_angle[i]);
-    }
+    Eigen::Vector2d zero_to_index_angle;
+    zero_to_index_angle[0] = 0.354043;
+    zero_to_index_angle[1] = 0.243433;
```

Comment:
Just a comment on naming: The *home offset* is actually the "index to zero" angle, not "zero to index" (i.e. zero = home + offset).

Reply:
Oo! so these number are actually super false! This is why I need the go_to after the calibration,
I will implement the go_to in another PR in the blmc_joint_modules.